src/cascadia/TerminalCore/terminalrenderdata.cpp

```diff
@@ -199,7 +199,7 @@ til::CoordType Terminal::_ScrollToPoints(const til::point coordStart, const til:
         _NotifyScrollEvent();
     }
 
-    return _scrollOffset;
+    return _VisibleStartIndex();

```

Comment:
Should we also update the comment, it currently reads:
```cpp
// Return Value:
// - The updated scroll offset
```

BTW, as someone new to this repo I have a doubt: what's the mutable viewport? and how does it differ from the viewport that `_scrollOffset` represents? 

Not the best place to ask questions, so I'm sorry 😅

Reply:
No worries! I probably didn't do the best job with that particular abstraction, and it bites even me to this day.

![image](https://github.com/microsoft/terminal/assets/18356694/e21445f3-2fad-4c15-b7a7-c1ab4158edbb)

(I should commit that somehere)

Reply:
PUT THIS IN THE REPO SOMEWHERE